dockerfiles/Dockerfile_linux_glibc

```diff
@@ -1,5 +1,12 @@
 FROM --platform=linux/amd64 050879227952.dkr.ecr.us-west-1.amazonaws.com/confluentinc/cli-centos-base-amd64:latest
 
+ARG GO_VERSION
+
+RUN wget "https://go.dev/dl/go$GO_VERSION.linux-amd64.tar.gz" && \
+    sudo tar -C /usr/local -xzf "go$GO_VERSION.linux-amd64.tar.gz"
+
+ENV PATH=${PATH}:/usr/local/go/bin:/root/go/bin
```

Comment:
If we `cd cli` first, then we shouldn't have to pass `$GO_VERSION`. We could just use `cat .go-version` instead.

Reply:
Might want to look into cloning `goenv` so the setup is the same as in the CLI?

Reply:
I don't think we need the setup to be that similar, but that's a good point about the `.go-version`, I'll make that change.